js/Main.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
```

Comment:
I suggest you to create class `Game` with public methods
Start and stop that will include `Field` and all required parts
Also it may have public methods `start` and `finish` to control the game